packages/asset/src/Asset.ts

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    html,
+    LitElement,
+    CSSResultArray,
+    TemplateResult,
+    property,
+} from 'lit-element';
+
+import styles from './asset.css.js';
+
+const file = (): TemplateResult => html`
+    <svg viewBox="0 0 128 128" class="file">
```

Comment:
Does this include all of the appropriate `aria-*` tags?

Reply:
I'm guessing we'll need clients to pass in a label for these null states, similar to what we do for `sp-icon`.

I'm assuming the alt tag for child <img> tags will work.